src/main/java/com/hedera/cli/hedera/crypto/CryptoTransferMultiple.java

```diff
@@ -188,25 +184,28 @@ private String promptPreview(AccountId operatorId, String jsonStringSender, Stri
                 + "\nyes/no");
     }
 
-    private void executeCryptoTransferMultiple(Client client, AccountId senderAccountID, AccountId operatorId,
+    private void executeCryptoTransferMultiple(Hedera hedera, Client client, AccountId senderAccountID, AccountId operatorId,
```

Comment:
A `client` object is already encapsulated within the `hedera` object. 

So it's probably not necessary for `executeCryptoTransferMultiple` or other similar functions to accept both as arguments?